output/cloud/expv2/pbcloud/metric.proto

```diff
@@ -21,7 +21,14 @@ enum MetricType {
 // that is encoded and sent over the wire.
 message MetricSet {
   // Required.
-  repeated Metric metrics = 1;
+  string test_run_id = 1;
+
+  // Required.
+  // time bucket size in seconds
+  uint32 aggregation_period = 2;
+
+  // Required.
+  repeated Metric metrics = 3;
```

Comment:
>I can see why moving these fields is a good thing, but I think we should rename the MetricSet type. It doesn't contain just metrics anymore, so the name is misleading.

I'll talk about this name internally and see what alternatives we'll be able to come up with.

>Maybe SampleContainer would be more accurate, and it would align with our internal name for it.

The idea was quite the opposite. We would like to align with the OpenMetrics/Prometheus protocol more than k6 internals. In this way, an eventual transition to the Prometheus remote write should be easier.

Also, just to make it clear, the `SampleContainer` of k6 is a group of raw samples, instead, this protocol doesn't expect raw samples. The _samples_ here are grouped by multiple layers:
- time: the bucket aggregation
- series: the `k6.metrics.TimeSeries`

Reply:
Based on the internal discussion, there is a consensus to keep `MetricSet`. The criteria is that the fields are attributes of the MetricSet adding a more defined context for the set of metrics.